dashboard/app/util_test.go

```diff
@@ -231,9 +246,25 @@ func (c *Ctx) setNoObsoletions() {
 	}
 }
 
+func (c *Ctx) decommissionManager(ns, oldManager, newManager string) {
+	c.transformContext = func(c context.Context) context.Context {
+		newConfig := replaceManagerConfig(c, ns, oldManager, func(cfg ConfigManager) ConfigManager {
+			cfg.Decommissioned = true
+			cfg.DelegatedTo = newManager
+			return cfg
```

Comment:
For consistency with replaceNamespaceConfig pass a pointer to ConfigManager to the callback and remove return value.